targets/TARGET_STM/TARGET_STM32L4/mbed_overrides.c

```diff
@@ -32,6 +32,10 @@ void mbed_sdk_init()
 {
     // Update the SystemCoreClock variable.
     SystemCoreClockUpdate();
```

Comment:
Hi @monkiineko, does HAL_Init need the clock speeds to be set before it is called? Just want to double check before I go through and remove this.